### DISCUSSION TITLE
大佬，脚手架怎么搞

### DISCUSSION THREAD

**Discussion by Atw-Lee** [alitajs/alita#555] [General]
无意间发现了这个仓库，我也想向alita一样搞一个脚手架。pnpx create-alita

我要如何入手呢，网上的文章阅读不太友好，有什么值得推荐的文章或视频吗。❤️

**Comment by xiaohuoni** [comment]
其实自己试着写一个掌握的最快，我正在写类似的 vue 版本的方案，你感兴趣的话，可以看看。感兴趣的话，可以先加下我微信 yu_xiaohu

文章的话，感觉我在掘金上的文章很适合入门？https://juejin.cn/column/7084812367116107789

**Reply by Atw-Lee** [reply]
thanks❤️
